RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -1675,7 +1675,7 @@ boolean L13_towerNSFinal()
 
 	if((pulls_remaining() == -1) || (pulls_remaining() > 0))
 	{
-		if(can_equip($item[Oscus\'s Garbage Can Lid]))
+		if(canPull($item[Oscus\'s Garbage Can Lid]))
```

Comment:
does this still need to do auto_can_equip()?

Reply:
Good call, it does. Added that to halibut as well